src/Umbraco.Web/Editors/PasswordChanger.cs

```diff
@@ -120,21 +120,24 @@ public PasswordChanger(ILogger logger, IUserService userService, HttpContextBase
                 return Attempt.Fail(new PasswordChangedModel { ChangeError = new ValidationResult("Password cannot be changed without the old password", new[] { "oldPassword" }) });
             }
 
-            if (passwordModel.OldPassword.IsNullOrWhiteSpace() == false)
+            //is the old password correct?
+            var backOfficeIdentityUser = AutoMapper.Mapper.Map<BackOfficeIdentityUser>(savingUser);
```

Comment:
I think to be safe and to have less dependencies outside of this method, we should do `var backOfficeIdentityUser = await userMgr.FindByIdAsync(savingUser.Id);` instead of the mapping logic here

Reply:
@Shazwazza you're a hero! Thanks! I was looking for a way to get the user instead of mapping it... I should have thought of "Find" 😄 I'll update the PR.